services/venice-server/src/main/java/com/linkedin/venice/server/VeniceServer.java

```diff
@@ -310,14 +312,20 @@ private List<AbstractVeniceService> createServices() {
         ? new RocksDBMemoryStats(metricsRepository, "RocksDBMemoryStats", plainTableEnabled)
         : null;
 
+    boolean whetherToRestoreDataPartitions = !isIsolatedIngestion()
```

Comment:
@sixpluszero , can you confirm this to be right?

I think this is PR we maybe shouldn't bother with this.  The original code has this hard coded to true, i think we should just pass true instead of potentially altering the behavior as this juncture.